torchdata/_extension.py

```diff
@@ -0,0 +1,45 @@
+import os
+import warnings
+from pathlib import Path
+
+import torch
+from torchdata._internal import module_utils as _mod_utils  # noqa: F401
+
+_LIB_DIR = Path(__file__).parent / "lib"
```

Comment:
Removing `"lib"` here may solve the issue. But, it's kind interesting that why it's working on linux/macos.